### DISCUSSION TITLE
Should GoogleContainerTools/kpt-functions-catalog be moved to kptdev/functions-catalog

### DISCUSSION THREAD

**Discussion by zevisert** [kptdev/kpt#4157] [Polls]
It seems like the [functions catalog repo](https://github.com/GoogleContainerTools/kpt-functions-catalog) is not getting much attention. Would optics<sup>1</sup> improve by moving it to this organization?

<sup>1: thus inviting more of the community to contribute</sup>

**Comment by CsatariGergely** [comment]
Took a while, but it is here now: https://github.com/kptdev/krm-functions-catalog
